pkg/controller/kubedirectorcluster/members.go

```diff
@@ -238,8 +253,43 @@ func handleReadyMembers(
 	cr *kdv1.KubeDirectorCluster,
 	role *roleInfo,
 	configmetaGenerator func(string) string,
+	connChg bool,
 ) {
 
+	shared.LogInfo(
+		reqLogger,
+		cr,
+		shared.EventReasonCluster,
+		fmt.Sprintf("In handle ready members, Conn chg: %t", connChg),
+	)
+
+	// shared.LogInfo(
+	// 	reqLogger,
+	// 	cr,
+	// 	shared.EventReasonCluster,
+	// 	fmt.Sprintf("In sync members, Annotations chg: %s", cr.Annotations[shared.ConnectionsChanged]),
+	// )
+	var connectionsVersion int64
+
+	if connectionsVersionStr, ok := cr.Annotations[shared.ConnectionsIncrementor]; ok {
+		connectionsVersion, _ = strconv.ParseInt(connectionsVersionStr, 10, 64)
```

Comment:
wondering what happens if someone tampers with annotation and either deletes it Or put a nonint value . We shoud test that scenario